### PR TITLE
陈思芹Week2作业

### PR DESCRIPTION
**Pull request by Sunlly0**
### What test did you complete before submitting the PR?

### CODE REVIEW

File: Week2-Data-Structure-I/homework/chensiqin/hashs.cpp [hunk 1]
```diff
@@ -0,0 +1,202 @@
+// #pragma once
+#include "hashs.h"
+
+
+int stringToInt(const string &s) {
```
Comment (l-iberty) [inline Week2-Data-Structure-I/homework/chensiqin/hashs.cpp]: string和int的转换直接用库函数即可：atoi/std::to_string

**Review by l-iberty — CHANGES_REQUESTED** [review]
删除exe文件

File: Week2-Data-Structure-I/homework/chensiqin/bloom_filter.cpp [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    int m=-n*log(e)/(log(2)*log(2));
+    int k=m*log(2)/n;
+    bs_size=m;
+    hash_size=k;
+    if(k>8){
+        hash_size=8;
+    }
+};
+
+int bloom_filter::insert(string key){
```
Comment (l-iberty) [inline Week2-Data-Structure-I/homework/chensiqin/bloom_filter.cpp]: 参数改为const string &key，下同

**Comment by Sunlly0** [comment]
2021.9.28 csq 优化week2作业

**Pull request closed without merging**